### PR TITLE
Added The Odin Project badge 

### PR DESCRIPTION
**Pull request by FFFF-0000h**

### CODE REVIEW

**Comment by Artymanprince** [comment]
This PR is based on the Badge Request made in issue #335.

**Comment by mehtaAnsh** [comment]
@Ileriayo should we add badges these long? This will completely disturb our interface. Let me know

@Artymanprince thanks for your contribution :D

**Comment by Ileriayo** [comment]
@Artymanprince, we need a more optimized badge URL.

File: README.md [hunk 1]
```diff
@@ -266,6 +266,7 @@ Copy the appropriate `![Name](link)` element and paste it in your Markdown file
...
...Gaf+fNAnCGIAwsgH32QAOAfBP4ZkdYCYC9E/Sb4NoEWAIApw3sJuCMw40CcA+DHAtIbgLEwTeBYDeAZ2ugGoB9JwDrWYAp+sgg/L4NkGcCgF/cILpGAUsF4PEgnNQGzDTsDALUaht074nsZwFHg3BCGzDTsDMK8IYBuBuBfcmTmwARcxSAgQKanQMGvt8TwXAKSPNAIgKnDawmYCug2VF4cKcgfwyCJAvwjwT2LfF2KND2iZC9EbAyPgHAc8HgWIg3DeBoK5zugYEJrAeD2zTY9qlwfCOweGCaBRgAtglYO8K2j8UHsqcgew4+BcA+FAq2A43fHT60gOQmQASOAlYTMDXQ+L3BgSI7QdMFIREAOn3QnYabfz5g2ErAboKEiBaPTwDCUYiZG2TNPyEy0C4ARq4HpgLwjgQItlEIbvCQFF/nYPAtINkD/TZgJgGETAHUflI0qICTTcC4IPkmkCuA2g9Lh1MAnfLAoA3s0/AtnhYfKGYBCCwdADLmADDHQjd5YYKvh0H7VpjOeaDfBrZJ4CbvDA0Uq4ATFuDvBrZhcLgJgMGdgzMAxE3gNi9ODgTOw39LCcBpAO4RQb4ALnh73AVAmAfAa4R3+oAC+haAZzyQiDwFIGMMhhsB4IEFwEkAngRgYvxOAAac4AsPJAJ3ECCYvhDAFQC4ZwF4ygJCF0RG9wIwAH7RBIiQwf6KABHRNwK4BID97BMinrQAA8C6Z3pjAJgBgDwTvDWA5TFxOgmA3RwAfdUEQhekewM42wQiAPjN2iBlHwYUcMEWSgCzAOB9AQzfNQEicBUw3M0DrCrIaQLeMPxVD7gKgBFBlge6Lvhl/pcBWF8kzgNgmcC3l3EdgPWTQjkWsLcBPtwYgLlfngHADMM/uIZLAQwcEeB8BRABIPwm/2sBDDxjDDIuyIc/AGAYKAsAMvzR718OYCDMAUDDnwEwUE4MfwgA/8cBZOVPvJfA3wGQoYHfrX8TAM4i+GX6jQA4g+C36TcDIBXBr9NvCEAKgt+n3xSAgb9HwEvk3xSAtz2Rl/nB1gAcqqBU/s0BiCMYykV7AEI35MM/BsBFUDT9VgHsCAqn3y6ABUHx9FsGMAwV0m8bwNABdAAdQAfQAbQKoEePHj169OjRo0ePHj169OjRo0ePHj169OjRo0ePHn8m/gd5Gp9rXkbGIwAAAABJRU5ErkJggg==)`
```
Comment (Ileriayo) [inline README.md:269]: URL is too long

**Review by Ileriayo — CHANGES_REQUESTED** [review]

**Comment by kohasummons** [comment]
I am closing this PR and Opening an Issue for the badge In question. Thanks everyone for the comments and participation. 

Issue: #432

**Pull request closed without merging**